src/admin/components/UpgradePopup.vue

```diff
@@ -0,0 +1,195 @@
+<template>
+    <div id="dokan-upgrade-wrapper">
+        <div id="dokan-upgrade-popper">
+            <p>{{ __("You're using", 'dokan-lite' ) }} <span>{{ __( 'Dokan Lite', 'dokan-lite' ) }}</span>. {{ __( 'To unlock more features, consider', 'dokan-lite' ) }}</p>
+            <a href="https://wedevs.com/dokan-lite-upgrade-to-pro" class="button">{{ __( 'Upgrade to Pro', 'dokan-lite' ) }}</a>
+        </div>
+
+        <div v-if="showPopup" id="dokan-upgrade-popup">
```

Comment:
Do we need this modal popup? if yes then how it will trigger? if no then we can remove the codes.